lib/spdy-transport/protocol/http2/parser.js

```diff
@@ -386,6 +419,28 @@ Parser.prototype.onSettingsFrame = function onSettingsFrame(header,
       settings[name] = value;
   }
 
+  if (constants.settingsIndex[constants.settings_server.SETTINGS_ENABLE_PUSH] in settings) {
```

Comment:
Perhaps you may want to use just a string value from `settingsIndex` directly? Instead of doing two lookups? Please consider not using ` in ` operator too, `settings['...'] !== undefined` should be fine.